spec/arns_spec.lua

```diff
@@ -574,10 +576,68 @@ describe("arns", function()
 				intent = "Extend-Lease",
 				years = 2,
 				name = "test-name",
+				currentTimestamp = timestamp + constants.oneYearMs,
 			}
 			_G.DemandFactor.currentDemandFactor = demandFactor
 			assert.are.equal(expectedCost, arns.getTokenCost(intendedAction))
 		end)
+		it("should throw an error if trying to increase undername limit an a record in the grace period", function()
```

Comment:
typo: `and`

Reply:
addressed in [da8ab39](https://github.com/ar-io/ar-io-network-process/pull/156/commits/da8ab39db7266c47d5652789cbbb17c4af73464b) and [a2bf822](https://github.com/ar-io/ar-io-network-process/pull/156/commits/a2bf8227d1ca5341b12c161c34eaf7ab35628ab2)